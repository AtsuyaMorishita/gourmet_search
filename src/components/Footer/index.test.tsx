import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import { Footer } from ".";

describe("footer", () => {
  it("render correctly", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();

    const link = screen.getByRole("link");
    expect(link).toHaveTextContent("ホットペッパー Webサービス");
  });
});
