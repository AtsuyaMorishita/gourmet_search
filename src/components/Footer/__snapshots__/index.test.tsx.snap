// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer render correctly 1`] = `
<div>
  <footer
    class="py-2 text-center text-[10px] lg:text-[14px] bg-[#FEF6E8] border border-[#F8E6CC] fixed bottom-0 left-0 w-[100%]"
  >
    Powered by
     
    <a
      class="text-[#017D01]"
      href="http://webservice.recruit.co.jp/"
      target="_blank"
    >
      ホットペッパー Webサービス
    </a>
  </footer>
</div>
`;
